src/Uno.UWP/UI/Colors.cs

```diff
@@ -35,21 +33,153 @@ public static Color Parse(string colorCode)
 				}
 				else
 				{
-					if (!_colorMap.TryGetValue(colorCode, out var color))
+					uint color = colorCode switch
 					{
-						var property = typeof(Colors).GetProperty(colorCode);
-
-						if (property != null)
-						{
-							_colorMap[colorCode] = color = (Color)property.GetValue(null)!;
-						}
-						else
-						{
-							throw new InvalidOperationException($"The color {colorCode} is unknown");
-						}
-					}
+						nameof(Transparent) => 0x00FFFFFF,
```

Comment:
The original behavior was that `colorCode` is case insensitive. Now it is case sensitive. Is that intended?

Reply:
good catch! I'll remote the `nameof` and compare with a lowercase string